### ISSUE TITLE
Menu titles

### ISSUE THREAD

**Issue by jennystratman** [community-table-market/addison-county#16]
Should be able to edit menu titles without deleting and starting over. Sometimes forget details or want to add them later.

**Comment by justindoran** [comment]
Yeah, that's a very nice feature and very straightforward to implement. Took care of that and enabled it in the latest push (Aug. 27). Let me know if you think it needs anything else.
